### ISSUE TITLE
GitHub Issue Template

### ISSUE THREAD

**Issue by frank113** [vincentarelbundock/pycountrycode#9]
# Summary

Packages sometimes employ custom templates for features and / or bugs. A minimally viable template would include the following:

+ Description of feature / problem
+ Checklist of items to remind the user to check their code prior to merging. 

# Considerations

+ I'll defer to @vincentarelbundock if this is needed for a package of this scope. If a template is not needed I can close the issue.

Thanks! :)

**Comment by vincentarelbundock** [comment]
Yeah, I think that's a good thing in general. Seems a bit low priority, but if you or someone else wants to write one I'd be happy!
